### ISSUE TITLE
"AttributeError: 'FlakesChecker' object has no attribute 'ANNASSIGN'" when using Python 3.6+ variable annotation

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#548]
In GitLab by @jstasiak on Dec 28, 2016, 17:10

I'm not sure whether this is supposed to go here or in the pyflakes repository but here we go:

*Please describe how you installed Flake8*

```
# in a virtualenv
$ pip install flake8
```

*Please provide the exact, unmodified output of `flake8 --bug-report`*

```
% flake8 --bug-report
{
  "dependencies": [
    {
      "dependency": "setuptools",
      "version": "28.8.0"
    }
  ],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.6.0",
    "system": "Darwin"
  },
  "plugins": [
    {
      "plugin": "import-order",
      "version": "0.11"
    },
    {
      "plugin": "mccabe",
      "version": "0.5.3"
    },
    {
      "plugin": "pycodestyle",
      "version": "2.2.0"
    },
    {
      "plugin": "pyflakes",
      "version": "1.3.0"
    }
  ],
  "version": "3.2.1"
}
```

*Please describe the problem or feature*

Flake8 crashes with: AttributeError: 'FlakesChecker' object has no attribute 'ANNASSIGN'. Full stack trace:

```
% flake8 test.py 
Process Process-1:
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.0/Frameworks/Python.framework/Versions/3.6/lib/python3.6/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/local/Cellar/python3/3.6.0/Frameworks/Python.framework/Versions/3.6/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/me/ve/36/lib/python3.6/site-packages/flake8/checker.py", line 666, in _run_checks_from_queue
    checker.run_checks(results_queue, statistics_queue)
  File "/Users/me/ve/36/lib/python3.6/site-packages/flake8/checker.py", line 606, in run_checks
    self.run_ast_checks()
  File "/Users/me/ve/36/lib/python3.6/site-packages/flake8/checker.py", line 510, in run_ast_checks
    checker = self.run_check(plugin, tree=ast)
  File "/Users/me/ve/36/lib/python3.6/site-packages/flake8/checker.py", line 464, in run_check
    return plugin['plugin'](**arguments)
  File "/Users/me/ve/36/lib/python3.6/site-packages/flake8/plugins/pyflakes.py", line 74, in __init__
    withDoctest=with_doctest)
  File "/Users/me/ve/36/lib/python3.6/site-packages/pyflakes/checker.py", line 494, in __init__
    self.handleChildren(tree)
  File "/Users/me/ve/36/lib/python3.6/site-packages/pyflakes/checker.py", line 816, in handleChildren
    self.handleNode(node, tree)
  File "/Users/me/ve/36/lib/python3.6/site-packages/pyflakes/checker.py", line 862, in handleNode
    handler = self.getNodeHandler(node.__class__)
  File "/Users/me/ve/36/lib/python3.6/site-packages/pyflakes/checker.py", line 696, in getNodeHandler
    self._nodeHandlers[node_class] = handler = getattr(self, nodeType)
AttributeError: 'FlakesChecker' object has no attribute 'ANNASSIGN'
```

*If this is a bug report, please explain with examples (and example code) what you expected to happen and what actually happened.*

The issue can be reproduced by running flake8 on the following piece of code: `a: int = 1`.

**Comment by asottile** [comment]
In GitLab by @johnthagen on Jan 14, 2017, 10:04

I believe this has been fixed upsteam in `pyflakes`: https://github.com/PyCQA/pyflakes/commit/dae4fa70d8d5fd98e81ceb72c007c0c60725dfec

I think `flake8` just needs to bump it's supported versions of `pyflakes`.

**Comment by asottile** [comment]
In GitLab by @jstasiak on Jan 15, 2017, 07:55

Thank you, I can confirm the issue is resolved with pyflakes 1.5.0. There are few remaining issues (https://github.com/PyCQA/pycodestyle/issues/575 covers at least one of them) but that's a different story:

```python
# test.py
from typing import Any, MutableMaping


def fun() -> Any:
    a: MutableMapping[str, int] = {}
    return a
```

Result with pyflakes 1.5.0:

```
% flake8 test.py
test.py:1:1: F401 'typing.MutableMaping' imported but unused
test.py:5:6: E701 multiple statements on one line (colon)
```

**Comment by asottile** [comment]
In GitLab by @jstasiak on Jan 15, 2017, 07:55

closed
